ssz/hash.py

```diff
@@ -40,10 +39,14 @@ def merkle_hash(input_items: Sequence[Any]) -> Hash32:
         items_per_chunk = SSZ_CHUNK_SIZE // len(input_items[0])
 
         # Build a list of chunks based on the number of items in the chunk
-        chunks = tuple(
+        chunks_unpadded = (
             b''.join(input_items[i:i + items_per_chunk])
             for i in range(0, len(input_items), items_per_chunk)
         )
+        chunks = tuple(
+            chunk.ljust(SSZ_CHUNK_SIZE, b"\x00")
+            for chunk in chunks_unpadded
+        )
```

Comment:
My instinct tells me it's faster in one for loop:
```python
@to_tuple
def get_chunkz_new(input_items, items_per_chunk):
    for i in range(0, len(input_items), items_per_chunk):
        yield b''.join(input_items[i:i + items_per_chunk]).ljust(SSZ_CHUNK_SIZE, b"\x00")
```

(Although maybe not much performance improvement difference.)